cdc/kv/region_worker.go

```diff
@@ -375,13 +375,14 @@ func (w *regionWorker) evictAllRegions(ctx context.Context) error {
 	w.statesLock.Lock()
 	defer w.statesLock.Unlock()
 	for _, state := range w.regionStates {
-		log.Info("on region fail fired", zap.Reflect("state", state))
+		state.lock.RLock()
```

Comment:
I prefer the following, in case `onRegionFail` blocks.

```
		state.lock.RLock()
		singleRegionInfo := state.sri
		state.lock.RUnlock()
```

Reply:
`onRegionFail` is non-blocking way
Besides, this code will be rewritten in later PR